community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -463,6 +463,15 @@ variable "reservation_name" {
   }
 }
 
+variable "future_reservation" {
+  description = <<-EOD
+  Allows for the use of future reservations. Input can be either the future reservation name or its selfLink in the format '[.../projects/PROJECT_ID/zones/ZONE/futureReservations/FUTURE_RESERVATION_NAME]'
```

Comment:
The format definition is not clear to me. 
What `...` stands for?
`[]` usually (shell args lingo) means optional.

Reply:
wrote it to mean you can optionally include the `https://google/compute/beta...` (however it goes), but considering we use regex anyway I'll just remove this portion